content/en/tracing/connect_logs_and_traces/ruby.md

```diff
@@ -17,17 +17,20 @@ further_reading:
       text: 'Correlate request logs with traces automatically'
 ---
 
-## Automatic Trace ID injection
+## Automatic log injection
```

Comment:
This header is misleading for Ruby. We do not support automatic log injection; all implementation is manual. The only "automatic" part of it is that we provide the data (trace ID, etc.) via a convenient API. Regardless, users must manually configure their loggers.

I don't have any suggestions to offer, other than that we handle the explanation of log correlation differently within the gem itself (more use case driven), but I suspect discussing broader changes within this documentation is out of scope. Food for thought nonetheless.

Reply:
Thanks @delner! Received this feedback yesterday, working on a fix for this 👍